docs/data/tree-view/simple-tree-view/customization/customization.md

```diff
@@ -9,24 +9,61 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
 
 # Simple Tree View - Customization
 
-<p class="description">Customize the rendering of your Tree View.</p>
+<p class="description">Learn how to customize the Tree View component.</p>
 
-## Custom icons, border and animation
+## Basics
 
-{{"demo": "CustomizedTreeView.js"}}
+### Custom icons
 
-## ContentComponent prop
+Use the `defaultCollapseIcon`, `defaultExpandIcon`, and `defaultEndIcon` props to customize the Tree View icons.
+You can use either icon libraries, such as [Material Icons](/material-ui/material-icons/), or create your own.
 
-You can use the `ContentComponent` prop and the `useTreeItem` hook to further customize the behavior of the TreeItem.
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
 
-Such as limiting expansion to clicking the expand icon:
+### Custom toggle animations
 
-{{"demo": "IconExpansionTreeView.js", "defaultCodeOpen": false}}
+Use the `TransitionComponent` prop on the `TreeItem` to pass a component that handles your animation.
+
+The below demo animates Material UI's [Collapse](/material-ui/transitions/#collapse) with [react-spring](https://www.react-spring.dev/).
+
+{{"demo": "CustomAnimation.js", "defaultCodeOpen": false}}
+
+### Custom styling
+
+Use the `treeItemClasses` to target internal elements of the Tree Item component and change their styles.
+
+{{"demo": "CustomStyling.js"}}
+
+### Adding custom content
+
+Use the `ContentComponent` prop and the `useTreeItem` to replace the Tree Item content with an entirely custom component.
+The demo below adds an avatar and custom typography.
+
+{{"demo": "CustomContentTreeView.js"}}
+
+## Common examples
 
-Or increasing the width of the item state indicator to be full-width:
+### Connection border
 
-{{"demo": "BarTreeView.js", "defaultCodeOpen": false}}
+Target the `treeItemClasses.group` class to add connection borders between the Tree View items.
 
-## Gmail clone
+{{"demo": "BorderedTreeView.js", "defaultCodeOpen": false}}
+
+### File explorer
+
+The demo below shows many of the previous customization examples brought together to make the Tree View component look completely different than its default design.
+
+{{"demo": "CustomizedTreeView.js"}}
+
+### Gmail clone
+
+Google's Gmail side nav is potentially one of the most famous tree view components.
+The demo below shows how to replicate it.
 
 {{"demo": "GmailTreeView.js"}}
+
+### Limit expansion to icon container
+
+The demo below shows how to use a custom content for the Tree View, and limit the expansion of the Tree Items to the `iconContainer`
```

Comment:
```suggestion
The demo below shows how to trigger the expansion interaction just by clicking on the icon container instead of the whole Tree Item surface.
```